doc/en/release/rest.html

```diff
@@ -0,0 +1,257 @@
+<!doctype html>
+<html>
+  <head>
+    <title>GeoServer Coverages</title>
```

Comment:
Seems like this is broader than just coverages.

Reply:
Good catch, I stole one of the generated files as an example.